core/frontend/src/render/primitives/mesh/RealityMeshPrimitive.ts

```diff
@@ -0,0 +1,59 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+/** @packageDocumentation
+ * @module Rendering
+ */
+
+import { QParams2d, QParams3d, RenderTexture } from "@bentley/imodeljs-common";
+import { GltfMeshData } from "../../../tile/internal";
+import { RenderMemory } from "../../RenderMemory";
+import { Mesh } from "./MeshPrimitives";
+
+export interface RealityMeshProps {

```

Comment:
What is the distinction between the classes in this file and the ones in RealityMeshPrimitive.ts?